kr.js

```diff
@@ -131,7 +131,7 @@ settings.chat.chatHeight 채팅 높이
 settings.chat.chatTimer 채팅 사라지기 타이머
 settings.chat.chatTextOutline 채팅 글자 테두리
 settings.chat.chatFilter 욕설 필터링
-settings.quality.header 성능
+settings.quality.header 고사양
```

Comment:
Since it is a header name, changing it to something broader other than "고사양" while preserving the original English meaning would be better.
Consider changing it to "품질."